go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/prometheus/common v0.40.0
 	github.com/stretchr/testify v1.8.2
 	golang.org/x/sys v0.5.0
-	google.golang.org/grpc v1.41.0
+	google.golang.org/grpc v1.53.0
```

Comment:
This needs to be tested properly to make sure it doesn't introduce something unexpected

Reply:
I have tested the following scenarios with datasource-http-backend plugin example:

- Plugin built with the new SDK and Grafana running the new SDK
- Plugin built with current stable SDK and Grafana running the new SDK
- Plugin built with new SDK and Grafana running the current stable SDK

All of them worked fine, so I think it is safe to bump it

Reply:
Awesome :rocket: